src/dune/context.mli

```diff
@@ -73,10 +73,10 @@ type t = private
   ; ocaml_bin : Path.t
   ; ocaml : Path.t
   ; ocamlc : Path.t
-  ; ocamlopt : Path.t option
-  ; ocamldep : Path.t
-  ; ocamlmklib : Path.t
-  ; ocamlobjinfo : Path.t option  (** Environment variables *)
```

Comment:
I guess the `(** Environment variables *)` comment was attached to `env : Env.t`.